### ISSUE TITLE
Incorrect error when using LoDash' parseInt via destructuring

### ISSUE THREAD

**Issue by MeoMix** [eslint/eslint#5639]
**What version of ESLint are you using?**

2.4.0. I'm using the latest `gulp-eslint`.

**What parser (default, Babel-ESLint, etc.) are you using?**

"babel-eslint": "^4.0.5",

**Please show your full configuration:**

```
"radix": [ 2 ],
```

**What did you do? Please include the actual source code causing the issue.**

```
import { parseInt } from 'lodash';
const result = parseInt('10');
```

**What did you expect to happen?**

Linting should pass. LoDash' `parseInt` implicitly uses base 10 as radix.

**What actually happened? Please include the actual, raw output from ESLint.**

I receive error message: `error    Missing radix parameter    radix`

**Comment by platinumazure** [comment]
Confirmed in ESLint demo with this test case:

```js
var parseInt;
parseInt('10');
```

I consider this a bug-- radix should be able to detect shadowing, similar to what we use in no-console and no-alert.

**Comment by mysticatea** [comment]
I'm working on this.
